examples/check-examples

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+for name in `cat maintained-notebooks.txt`
```

Comment:
Do we expect there to be unmaintained notebooks? Ideally not. Can we just test all the notebooks in the directory? Then people can't add one and forget to add it to `maintained-notebooks.txt`.

Reply:
Oh, yes, checking all notebooks is better. I'll take care of it.